src/lua/debug.lua

```diff
@@ -0,0 +1,8 @@
+local debug = require('debug')
+
+local function sourcepath()
+    return debug.getinfo(2, "S").source:sub(2):match("(.*/)") or './'
```

Comment:
BTW, why './', not '.'?

Reply:
Same thoughts. Decided to eliminate trailing `/`